package.json

```diff
@@ -10,7 +10,6 @@
         "bower": "~1"
     },
     "scripts": {
-        "postinstall" : "./node_modules/.bin/bower install",
```

Comment:
This line is removed because it't gonig to fail on npm 3 (which I'm using right now).

Npm 3 stores all dependenties in a [flat](https://docs.npmjs.com/how-npm-works/npm3) way, instead of a tree (what npm1 & npm2 does).

The problem is. If I'm installing cr-acl in my project as a dependency, the postinstall of cr-acl will try `./node_modules/.bin/bower install` which will fail because cr-acl has no node_moldules folder. the bower dependecy will be stored @ the root of the project.